src/plugins/intel_gna/src/gna_graph_tools.hpp

```diff
@@ -693,6 +706,49 @@ std::vector<std::pair<CNNLayerPtr, int>> CNNNetGetPrevLayersSkip(CNNLayerPtr ori
     return prevLayers;
 }
 
+/**
+ * @brief Removes 'to_remove' layer from between two other - 'prev' and 'next', then connects 'prev' and 'next'
+ * @param prev           Layer before 'to_remove'
```

Comment:
doxygen should only contain params which are in function prototype

Reply:
Corrected.